### ISSUE TITLE
power issues on Android 5.0

### ISSUE THREAD

**Issue by davidpanderson** [BOINC/boinc#1894]
from a user email:

Tablet: RCA
Model #RCT6773W22B
Android version 5.0
BOINC Version 7.4.53

Though I have preferences set to run on charger or with USB power
connection, and tablet is connected to charger (verified by seeing battery
is charging), BOINC continues with message "Connect your device to a charger
to continue computing." I've tried couple chargers, and connected via USB to
couple computers. However, continue with above message. 

Also note that when I add "allow use with battery (set to be 90% or
greater)" to preferences, I'll then get the message "Computing will resume
when battery charge reaches 90% (currently 100%)." Strange!  It MAY crunch
for a short time (no longer than 2-3 days) and then stop again after I
restart BOINC Mgr. or abort a WU, or maybe at its own whim.
 
Tablet otherwise works well on all other apps installed. Note that above
BOINC version is working well on phones with Android 4.1.1 and 6.01 (though
very slow crunching on 4.4.1 phone with 1 processor, as expected).

**Comment by brevilo** [comment]
Which version of Android is this exactly? Also, is this plain Android (AOSP) or some vendor "enhanced" flavor of it?

As usual: I can't reproduce this using `master` on AOSP 5.1.1 (LMY48M on `hammerhead`) , so this also should be fixed as soon as we get the next release out of the door...

**Comment by James-W2** [comment]
I initiated this request, which David Anderson passed on above.  

To answer the above questions:
Android version 5.0
Build #RCT6773W22B-ANDROID5.0-V06-V1.10.20
Kernel Version:
- 3.10.54 
- EM8127-7067A@V01M #1 
- Tue Jul 7 10:56:23 CST 2015

This should answer 1st question.  However, can't say for sure if "enhanced" version or not, though likely RCA customized somewhat.  Is there somewhere I can look if above doesn't give you any clues? I got above info from About Tablet section.

**Comment by brevilo** [comment]
Thanks for the update.

The only real way to get this fixed is to get a [fresh BOINC release](https://github.com/BOINC/boinc/issues/1846) out of the door. Unfortunately there's no ETA for it yet.

Cheers

PS: please note that your device is insecure as it can get and, IMHO, shouldn't be connected to the internet at all - the usual Android-device-without-updates-issue 😞

**Comment by James-W2** [comment]
FYI - I checked with RCA Support to see if they had an update from Android V5.0 for my tablet, and they stated: 

> Unfortunately, there are no available updates for the current firmware.

**Comment by brevilo** [comment]
Yep, sorry to hear that but it's the known Achilles' heel of the Android ecosystem. Buy a cheap device and your precious new toy is insecure (almost) on arrival. Even flagship devices suffer from this disease if you opt for the "wrong" manufacturer. Golden rule: no Android purchase without prior update policy and/or hackability research, if you care about device (your data!) security. Sad but true.

**Comment by James-W2** [comment]
By trial and error, of late I've noted the following: I've been running Seti@Home and Rosetta@Home on this device.  Of late, when I've noticed the device has stopped crunching due to noted power issues, I've aborted a Rosetta WU, and usually this will start the device crunching again.  For now I've selected no new tasks for Rosetta, and it has continued to process the Seti WUs for last couple days at least.  Don't know _why_ this would be, but wanted the folks involved with this issue to be aware.  Thanks again.
